include/glow/Graph/Graph.h

```diff
@@ -883,6 +883,12 @@ class Function final : public IRContainer {
   ReshapeNode *createFlatten(llvm::StringRef name, NodeValue input,
                              unsigned_t axis);
 
+  /// Flattens the input tensor into a 2D matrix. If input tensor has shape
+  /// (d_0, d_1, ... d_n) then the output will have shape:
+  /// ((d_axis X d_(axis+1) ... X d_n, d_0 X d_1 ... d_(axis-1)).
```

Comment:
Is this correct? Wouldn't it actually be something like:

```
/// (d_axis, (d_0 X d_1 ... d_(axis-1) X d_(axis+1) ... X d_n)).
```

Reply:
Done